docs/user-guide/docs-source/appendix-contributing.md

```diff
@@ -0,0 +1,98 @@
+# Appendix: Contributing to Ambit Development
+
+This section provides important information for anyone who would like to contribute to the development of the Ambit tools or to make custom modifications to Ambit for their own purposes. It covers technical setup and developer workflow. For general contributing guidelines and process, see the repository's [CONTRIBUTING.md file](https://github.com/aws-samples/aws-ambit-scenario-designer-ue4/blob/main/CONTRIBUTING.md).
+
+> ✏️ This guide assumes you already have Unreal Engine development experience.
+
+
+
+## Setup
+
+Ensure you have completed the main [Setup > Initial Setup (One-Time)](../setup/#initial-setup-one-time) instructions from this user guide.
+
+In addition to having Unreal Engine 4.27 installed, you will also need the Microsoft Visual Studio development tools required for UE4 C++ development. If you need help with these setup steps, refer to the Unreal Engine 4 documentation, especially ["Setting Up Visual Studio for Unreal Engine"](https://docs.unrealengine.com/4.26/en-US/ProductionPipelines/DevelopmentSetup/VisualStudioSetup/).
+
+Create a new Unreal Engine C++ project to use as a host project while you work on Ambit development. There are no special configuration requirements for this project. The "Blank" project template with default settings will work just fine, but you can also customize the settings if you'd like.
+
+Clone this repository into the "Plugins" folder of the UE project you created. Note, you will need to create the "Plugins" folder first.
+
+`<your sample project>\Plugins\aws-ambit-scenario-designer-ue4\`
+
+Once the Ambit plugin repository has been cloned, right-click on the `*.uproject` file at the root of the project directory and select the "Generate Visual Studio project files" option. Select the appropriate engine version if required. A new Visual Studio solution file (`*.sln`) will be generated for you.
+
+Launch your sample project by clicking on the `*.uproject` file. When prompted to rebuild missing modules, click "Yes".
+
+![Missing Modules prompt](images/appendix/RebuildMissingModules.png)
+
+Once the project opens, enable the "Functional Testing Editor" plugin, and restart (or simply close) the editor for the plugin to take effect. This plugin is used to run Ambit's automated tests.
+
+
+
+## Development Workflow
+
+Follow this workflow when making changes to the Ambit C++ code...
+
+1. Launch the project's `*.sln` file in Visual Studio. 
+2. In the **Solution Explorer** window, right-click on your project and select **Set as Startup Project**. 
+3. Make your code edits.
+4. Compile and run the solution through either of these methods:
+   1. To build only: Select `Build > Build Solution` from the Visual Studio menubar
+   2. To build and run: Press the **Local Windows Debugger** button (or press **F5**).
```

Comment:
The nested bullet points don't show up appropriately in the browser. Rather than being 
-4
----1
----2
-5

it shows up as 
-4
-5
-6
-7